### PR TITLE
update storybook to latest

### PR DESCRIPTION
**Pull request by joscha**
This updates storybook to the latest release and enables a few options. This PR is in preparation for a problem I found with storybook's `setOptions` call.

### CODE REVIEW

File: integration-tests/.storybook/config.js [hunk 1]
```diff
@@ -38,4 +38,5 @@ setOptions({
   showSearchBox: false,
   downPanelInRight: true,
   sortStoriesByKind: false,
+  hierarchySeparator: /\./,
```
Comment (joscha) [inline integration-tests/.storybook/config.js]: This is a new feature in `3.2`, which potentially touches the story groups - currently only in UI, but I think it is important we make it part of the test suite for the future.

File: integration-tests/package.json [hunk 2]
```diff
@@ -21,14 +21,14 @@
...
     "@percy-io/in-percy": "file:../packages/in-percy",
-    "@percy-io/react-percy": "^0.1.1",
-    "@percy-io/react-percy-storybook": "^0.1.8",
-    "@storybook/addon-actions": "^3.1.9",
-    "@storybook/addon-info": "^3.1.9",
-    "@storybook/addon-knobs": "^3.1.9",
-    "@storybook/addon-links": "^3.1.6",
-    "@storybook/addon-options": "^3.1.6",
-    "@storybook/react": "^3.2.0-alpha.12",
+    "@percy-io/react-percy": "file:../packages/react-percy",
```
Comment (joscha) [inline integration-tests/package.json]: These two were actually using the last published version for some reason.

File: package.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
     "build": "vitruvius build --src src --dest lib",
     "clean": "rimraf ./packages/*/lib",
     "lint": "eslint . --cache",
-    "postinstall": "npm run clean && npm run build",
+    "postinstall": "yarn run clean && yarn run build",
```
Comment (joscha) [inline package.json]: I think these were leftovers from moving to yarn.

File: integration-tests/.storybook/config.js (hunk 1 above)
Comment (timhaines) [inline integration-tests/.storybook/config.js]: @joscha We have some stories in the integration tests that are relying on the default hierarchySeparator.  I think that got broken in the 3.2.0 release, and was fixed in 3.2.1  (3.2.3 has shipped now)

The idea of using a custom is a good one - but we should probably update the story names to match.

**Comment by timhaines** [comment]
I'm curious to hear more about what your issue with `setOptions` was.  I know someone else who struck a problem with it.  Moving it to the bottom of the config file solved the issue they were having.

**Comment by joscha** [comment]
> Moving it to the bottom of the config file solved the issue they were having.

That is exactly it.

File: integration-tests/.storybook/config.js (hunk 1 above)
Comment (joscha) [inline integration-tests/.storybook/config.js]: 👍  Will change a few of them

**Comment by joscha** [comment]
> I'm curious to hear more about what your issue with setOptions was. I know someone else who struck a problem with it. Moving it to the bottom of the config file solved the issue they were having.

Added a note about it in f296dbcc195e05c60c40a9c47af2c69fdc1cda5b

File: integration-tests/.storybook/config.js (hunk 1 above)
Comment (joscha) [inline integration-tests/.storybook/config.js]: added 175a6ca

**Comment by joscha** [comment]
All changes in @timhaines - let me know if there is anything else needed

**Comment by timhaines** [comment]
Thanks for this @joscha -- I'll merge it when CI's finished running.

I think I'll add a note to our docs about `setOptions` tomorrow too, until that restriction is fixed.  If you notice it does get fixed, could you let me know please?

**Comment by joscha** [comment]
> If you notice it does get fixed, could you let me know please?

for sure!

**Pull request merged**